core/src/main/java/com/linecorp/armeria/client/circuitbreaker/CircuitBreakerMapping.java

```diff
@@ -31,6 +30,62 @@
 @FunctionalInterface
 public interface CircuitBreakerMapping {
 
+    /**
+     * Builder class for building a CircuitBreakerMapping based on a combination of host, method and path.
+     */
+    class Builder {
```

Comment:
- Could you move this builder class to top-level for keeping consistency with other builders?
- Add `static builder()` method for creating this builder?

Reply:
Done